config_coordinator/README.md

```diff
@@ -53,7 +53,7 @@ docker run --publish 8999:8999 --name coordinator --rm -v /my/storage:/var/lib/c
             config_coordinator -db /var/lib/coordinator/configs.json
 ```
 
- will cause Docker to mount the host machine directory `/my/storage` at the path `/var/lib/coordinator`
+ This will cause Docker to mount the host machine directory `/my/storage` at the path `/var/lib/coordinator`
  in the container, and to use a file named `configs.json` in that directory to persist its state.
 
  ## Copyright
```

Comment:
Should we not change the date of the Copyright?

Reply:
The copyright statements in the repo all need updating. @ddsharpe is going to update them after I submit my changes.